samples/betaFeatures.js

```diff
@@ -0,0 +1,586 @@
+/**
+ * Copyright 2017, Google, Inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This application demonstrates how to perform basic recognize operations with
+ * with the Google Cloud Speech API.
+ *
+ * For more information, see the README.md under /speech and the documentation
+ * at https://cloud.google.com/speech/docs.
+ */
+
+'use strict';
+
+// # DIARIZATION
+
+function speechTranscribeDiarization(speechFile) {
+  //[START speech_transcribe_diarization]
+  const fs = require('fs');
+
+  // Imports the Google Cloud client library
+  const speech = require('@google-cloud/speech').v1p1beta1;
+
+  // Creates a client
+  const client = new speech.SpeechClient();
+
+  /**
+   * TODO(developer): Uncomment the following lines before running the sample.
+   */
+  // const filename = 'Local path tols -a audio file, e.g. /path/to/audio.raw';
+  // const encoding = 'Encoding of the audio file, e.g. LINEAR16';
+  // const sampleRateHertz = 16000;
+  // const languageCode = 'BCP-47 language code, e.g. en-US';
+
+  const config = {
+    encoding: `LINEAR16`,
+    sampleRateHertz: 8000,
+    languageCode: `en-US`,
+    enableSpeakerDiarization: true,
+    diarizationSpeakerCount: 2,
+    model: `phone_call`,
+  };
+
+  const audio = {
+    content: fs.readFileSync(speechFile).toString('base64'),
+  };
+
+  const request = {
+    config: config,
+    audio: audio,
+  };
+
+  client
+    .recognize(request)
+    .then(data => {
+      const response = data[0];
+      const transcription = response.results
+        .map(result => result.alternatives[0].transcript)
+        .join('\n');
+      console.log(`Transcription: ${transcription}`);
+      console.log(`Speaker Diarization:`);
+      const chunks = response.results.map(result => result.alternatives[0]);
+      console.log(`WORDS: `);
+      //words.forEach(a => console.log(JSON.stringify(a, null, 4)));
+      const lastChunk = chunks[chunks.length - 1].words;
+      lastChunk.forEach(a => console.log(JSON.stringify(a, null, 4)));
```

Comment:
I don't see `word` and `speaker_tag` being printed

Please update this to update the [canonical `speech_transcribe_diarization ` sample](https://github.com/googleapis/nodejs-speech/blob/master/samples/recognize.js)